x-pack/plugins/event_log/generated/mappings.json

```diff
@@ -267,6 +267,40 @@
                         }
                     }
                 },
+                "alert": {
+                    "properties": {
+                        "rule": {
+                            "properties": {
+                                "execution": {
+                                    "properties": {
+                                        "uuid": {
+                                            "type": "keyword"
+                                        },
```

Comment:
Thanks for reviewing this PR, @spong 👍 

I also noticed some `"ignore_above": 1024` present in the schema, but I don't know why they were added. To me, it looks highly unlikely that rule status or UUID will ever exceed 1024 characters. And even if they do, shouldn't we index them anyway? 🤷‍♂️ That said, I don't mind adding `"ignore_above": 1024` if you feel strongly about it.

Reply:
We got bitten in the past (by a security dev!) who pasted some 3MB string into what would go in a keyword field (actually, I think it was nested, but keyword has the same issue), where the index was rejected because of the size.  `ignore_above` allows the index to succeed, just doesn't store more than that amount in the ES field (still available in the source).  So it seems like we should always use this attribute :-)

Reply:
I don't have a strong preference here as well. But would it bring any value?

https://www.elastic.co/guide/en/elasticsearch/reference/current/ignore-above.html

> Strings longer than the ignore_above setting will not be indexed or stored.

If we ever try to index a UUID with the length more than 1024 characters:

- it would probably indicate a bug in the code
- we won't notice it (if we don't validate events before logging) because the event would be indexed, silently ignoring the UUID
- missing UUIDs in some events might lead to bugs in the app in other places

On the other hand, this situation is unlikely.

Reply:
Okay, I see. Thanks for clarifying that 👍 Added `ignore_above` to all new `keyword` fields.